### DISCUSSION TITLE
Railway template

### DISCUSSION THREAD

**Discussion by X-lem** [CorentinTh/enclosed#174] [Show and tell]
I created a template on [Railway](https://railway.app) that anyone can use to self host this! One click (more or less) to deploy!

[![Deploy on Railway](https://railway.app/button.svg)](https://railway.app/template/0sqPWB)

**Comment by CorentinTh** [comment]
Hi @X-lem 
Thank you for taking the time to create the template, it's much appreciated! 🙏 I'll add the button to the documentations

However, I'd kindly ask that you avoid including referral codes in contributions to open-source projects in the future. It’s not really fair to add a referral code to someone else's open-source project. To keep things neutral, I’ve gone ahead and edited your message to remove the codes.

Thanks for your understanding!

**Reply by X-lem** [reply]
@CorentinTh Apologies, don't mean to offend. In the interest of transparency (and in case you're not familiar) all templates on Railway have a "kickback" program. People create templates from open source software/Docker images they like/use. They can publish these templates and a link is generated (that can include a referral code). Regardless of whether or not the referral code is included the original creator of the template will receive a kickback when the template is used by others. This is Railway's way of encouraging people to use/create templates and makes deploying open source software really simple. I personally use templates created by others (that were not build by the owner of the project). From my experience with Railway this is fairly common.

Railway docs for more info: https://docs.railway.app/reference/templates#kickback-program

**Reply by CorentinTh** [reply]
Hi @X-lem,

No hard feelings at all, thank you for taking the time to explain how Railway's kickback program works, I wasn't aware of that!

Perhaps it would be better if, as the creator of the project, I'm the one to create and maintain the official template. That way, we can keep everything consistent and transparent for enclosed users and selfhosters
